portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,70 +27,107 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-
+ 
 /** Servlet that returns a random quote. */
 @WebServlet("/data")
 public final class DataServlet extends HttpServlet {
+ 
+    /** Declared as class variable so that initialization happens once in init()
+        rather than everytime in doGET() */
+    List<String> funFacts = new ArrayList<>();
     
-    /** One time initialization may be used for future. */
-    // @Override
-    // public void init() {
-    // }
-
-
-    /** Merges dataFactsAndComments with pastComments to convert as one ArrayList into JSON to put on servlet. */
+    /** One time initialization. */
     @Override
-    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        List<String> funFacts = new ArrayList<>();
+    public void init() {
         funFacts.add("I enjoy all music genres especially EDM!");
         funFacts.add("I studied abroad in Barcelona :)");
         funFacts.add("My favorite food is rotisserie chicken!");
         funFacts.add("I spend a lot of days on the lake!");
+    }
+ 
+    /** Merges dataFactsAndComments with pastComments to convert as one ArrayList into JSON to put on servlet. */
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {        
+        // Add fun facts first to factsAndComments return variable.
+        List<String> factsAndComments = new ArrayList<>();
+        factsAndComments.addAll(funFacts);        
         
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
         Query query = new Query("siteComments").addSort("timeStamp", SortDirection.DESCENDING);
         PreparedQuery results = datastore.prepare(query);
-
+ 
         List<String> pastComments = new ArrayList<>();
+        int validNumRequest = 0;
+        int numCommentsWanted = 0;
+        boolean numCommentsWantedTrigger = true;
+ 
         for (Entity entity : results.asIterable()) {
-            String comment = (String) entity.getProperty("newComment");
-            pastComments.add(comment);
+            validNumRequest = Integer.parseInt((String)(entity.getProperty("seeNumComments")));
+            if (validNumRequest >= 0){
+                String comment = (String) (entity.getProperty("newComment"));
+
+                //For testing queried result comments
+                // System.out.println("Query result comment: " + comment);
```

Comment:
Haha - this is apparently the code review where I really nitpick all of your comments!

When you've commented out code, there shouldn't be a space after the "//".  When you're doing a normal comment, there should be a space after the "//".

I realize this is really trivial code review feedback. :p

Reply:
No problem! Good to know the (Googler) developer ways